docs/modules/SDDs/pages/getting-started.adoc

```diff
@@ -0,0 +1,265 @@
+kubectl -n lieutenant set env deployment/lieutenant-operator -c lieutenant-operator SKIP_VAULT_SETUP=true
+----
+
+=== Install Lieutenant API
+
+Install *Lieutenant API* with the following commands:
+
+[source,shell]
+----
+# API deployment
+kubectl -n lieutenant apply -k github.com/projectsyn/lieutenant-api/deploy
+
+# Ingress
+export K3S_INGRESS_IP=$(kubectl -n kube-system get svc traefik -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
+kubectl -n lieutenant apply -f -<<EOF
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: lieutenant-api
+spec:
+  rules:
+  - host: lieutenant.${K3S_INGRESS_IP}.nip.io
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: lieutenant-api
+          servicePort: 80
+EOF
+----
+
+Check that the API is accessible:
+
+[source,shell]
+----
+curl http://lieutenant.${K3S_INGRESS_IP}.nip.io/healthz
+----
+This should return `ok` as the answer to the `curl` command.
+
+TIP: The API documentation can be accessed in your browser under http://lieutenant.${K3S_INGRESS_IP}.nip.io/docs.
+
+=== Prepare Lieutenant Operator access to GitLab.com
+
+Create a Kubernetes secret which contains the access token for the GitLab.com API, which can be generated here: https://gitlab.com/profile/personal_access_tokens (needs `api` scope).
+
+Replace `MYTOKEN` with the generated GitLab.com API token.
+
+[source,shell]
+----
+export GITLABCOM_TOKEN=MYTOKEN
+kubectl -n lieutenant create secret generic vshn-gitlab \
+  --from-literal=endpoint=https://gitlab.com \
+  --from-literal=hostKeys="gitlab.com ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAfuCHKVTjquxvt6CM6tdG4SLp1Btn/nOeHHE5UOzRdf" \
+  --from-literal=token=${GITLABCOM_TOKEN}
+----
+
+NOTE: This secret currently needs to be named exactly like that. See https://syn.tools/lieutenant-api/deployment.html#_gitlab[syn.tools] and https://github.com/projectsyn/lieutenant-operator/issues/48[GitHub issue] for more information.
+
+=== Prepare Lieutenant API Authentication and Authorization
+
+As the Lieutenant API uses the underlying Kubernetes cluster for authentication and authorization, the following objects need to be created:
+
+* `Role`
+* `RoleBinding`
+* `ServiceAccount`
+
+[source,shell]
+----
+kubectl -n lieutenant apply -f -<<EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: lieutenant-api-user
+rules:
+- apiGroups:
+  - syn.tools
+  resources:
+  - clusters
+  - tenants
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: lieutenant-api-user
+roleRef:
+  kind: Role
+  name: lieutenant-api-user
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: api-access-synkickstart
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: api-access-synkickstart
+EOF
+----
+
+=== Create Lieutenant Objects: Tenant and Cluster
```

Comment:
"Create your first Lieutenant configuration objects using the API to test" ?

Reply:
added